src/pages/Home/EntriesList/Entry.tsx

```diff
@@ -47,95 +48,89 @@ const Image = styled.img<{ isFullWidth: boolean }>`
   ${({ isFullWidth }) => isFullWidth && 'width: 100%; height: 100%;'}
 `
 
-const handleEntryClick = async (
-  item: any,
-  setDetailsData,
-  setEntryStatus,
-  setItemId,
-  setEvidences,
-  setIsDetailsModalOpen
-) => {
-  const details = await fetchFromIPFS(item.data)
-  setDetailsData(details.values)
-  setEntryStatus(item.status)
-  setItemId(item.itemID)
-
-  // Fetch evidences
+interface IEntry {
+  item: GraphItem
+}
 
-  //const evidenceData = await item.requests[item.numberOfRequests - 1].evidences;
+const Status: React.FC<{
```

Comment:
I would abstract the interface, like this:

interface IStatus {
  status:
    | 'Registered'
    | 'Absent'
    | 'RegistrationRequested'
    | 'ClearingRequested'
  disputed: boolean
  bounty: string
}

then use it like:

const Status: React.FC<IStatus> ...